### DISCUSSION TITLE
How to get only published in Browser

### DISCUSSION THREAD

**Discussion by michaeljcoyne** [area17/twill#1277] [General]
How do I only get published records in the Browser form?

Is there documentation on this please?

**Comment by pboivin** [comment]
Hi @michaeljcoyne, I'm not sure this is fully documented yet, but it can be done by overriding `getBrowserItems()` in your controller :

```
class CategoryController extends BaseModuleController
{
    // ...

    protected function getBrowserItems($scopes = [])
    {
        $scopes['published'] = true;

        return parent::getBrowserItems($scopes);
    }
}
```
